src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  margin-right: auto;
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  flex-wrap: nowrap;
+}
+
+.nav__link {
+  position: relative;
+  flex-shrink: 0;
+  text-decoration: none;
+  height: 15px;
+  color: rgb(0, 0, 0);
+  font-weight: 500;
+  font-size: 12px;
+  margin: 22px 20px;
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 100%;
+  height: 4px;
+  bottom: -18px;
+  border-radius: 8px;
+  background-color: var(--blue-color);
+}
+
+@media (max-width: 1024px) {
+  .header {
+    padding: 0 20px;
+  }
+
+  .logo {
+    margin-right: 243px;
```

Comment:
The `margin-right` value of `243px` for the `.logo` class in the media query might cause layout issues on smaller screens. Consider using a more flexible layout approach, such as flexbox properties, to ensure the logo and navigation items are properly aligned across different screen sizes.